### PR TITLE
PLT-2123 Member show on Activate and Hide on deactivate from channel member li…

### PR DESCRIPTION
**Pull request by enahum**
…st and at_mention

### CODE REVIEW

File: api/user.go [hunk 1]
```diff
@@ -1510,6 +1510,8 @@ func UpdateActive(c *Context, user *model.User, active bool) *model.User {
 			RevokeAllSession(c, user.Id)
 		}
 
+		<-Srv.Store.Channel().ExtraUpdateByUser(user.Id, model.GetMillis())
+
```
Comment (coreyhulen) [inline api/user.go]: I think we should check for the error and set `c.Err; return` so it will get printed.

Comment (enahum) [inline api/user.go]: Sure thing, I'll do that.

Comment (enahum) [inline api/user.go]: I did avoid the `return` cause that action doesn't prevent the user for being updated.

**Comment by coreyhulen** [comment]
+1

**Comment by crspeller** [comment]
+1

**Pull request merged**